src/main/scala/tile/Core.scala

```diff
@@ -162,18 +162,18 @@ trait HasCoreIO extends HasTileParameters {
   implicit val p: Parameters
   def nTotalRoCCCSRs: Int
   val io = new CoreBundle()(p) {
-    val hartid = UInt(hartIdLen.W).asInput
-    val reset_vector = UInt(resetVectorLen.W).asInput
-    val interrupts = new CoreInterrupts().asInput
-    val imem  = new FrontendIO
-    val dmem = new HellaCacheIO
-    val ptw = new DatapathPTWIO().flip
-    val fpu = new FPUCoreIO().flip
-    val rocc = new RoCCCoreIO(nTotalRoCCCSRs).flip
+    val hartid = Input(UInt(hartIdLen.W))
+    val reset_vector = Input(UInt(resetVectorLen.W))
+    val interrupts = Input(new CoreInterrupts())
+    val imem  = Input(new FrontendIO)
+    val dmem = Input(new HellaCacheIO)
```

Comment:
I don't think this is right. It should just be `= new HellaCacheIO`

Reply:
My bad. Too sleepy when I was working on thid